contracts/utils/FlowLimit.sol

```diff
@@ -106,9 +113,11 @@ contract FlowLimit is IFlowLimit {
     function _addFlowOut(uint256 flowOutAmount) internal {
         uint256 flowLimit = getFlowLimit();
         if (flowLimit == 0) return;
```

Comment:
Do we want to keep track of flow in/flow out even if the flowLimit is infinite? In case it gets changed to a finite value after and previous flows affect the current allowance? This issue was brought up in code4rena.

Reply:
flow limit doesn't need to be applied retroactively, it should be applicable on future transfers. And doesn't make sense to pay the significant gas penalty for tokens with no flow limits anyways.